.eslintrc.json

```
@@ -28,14 +25,24 @@
     "react-refresh/only-export-components": [
       "warn",
-      { "allowConstantExport": true }
+      {
+        "allowConstantExport": true
+      }
     ],
     "react-hooks/rules-of-hooks": "error",
     "react-hooks/exhaustive-deps": "warn",
-    "@typescript-eslint/no-unused-vars": ["warn", { "argsIgnorePattern": "^_" }],
-    "@typescript-eslint/no-explicit-any": "warn",
+    "@typescript-eslint/no-unused-vars": [
```

Biome (2.4.11)

[error] The key `@typescript-eslint/no-unused-vars` was already declared.

(lint/suspicious/noDuplicateObjectKeys)